packages/clarity-js/src/clarity.ts

```diff
@@ -95,10 +95,24 @@ export function end(): void {
   }
 }
 
-export function tag(key: string, value: string): void {
+export function tag(name: string): void {
```

Comment:
So tags are now more like, well, normal tags that accumulate for a page? What if the same tag is used multiple times?

Reply:
We capture them in the events and we don't de-dupe at collection time. De-duping may happen on server but that's outside the scope of client instrumentation.